src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -366,7 +366,7 @@ export default class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang
     }
 
     forEach(elementRef.childNodes, function(child) {
-      height = height + (<HTMLElement> child).offsetHeight
+      height += (child as HTMLElement).offsetHeight
```

Comment:
Thanks @bzabos for reminding this, I've already seen such suggestion for Laravel (using collection rather than foreach), but didn't remember to apply it with JS, hihi.
Well noted ;)
I'll try to change these loops to start.